Examples/ConsoleApplication/Program.cs

```diff
@@ -160,42 +159,6 @@ static void Main()
             surveyFieldworkService.StartFieldworkAsync(newSurvey.SurveyId).Wait();
             surveyFieldworkStatus = surveyFieldworkService.GetStatusAsync(newSurvey.SurveyId).Result; //Should be started
 
-            // Example of a download data request: filtering testdata collected today
-            var surveyDataService = connection.GetService<INfieldSurveyDataService>();
```

Comment:
as there is a new call for the data download, it would be nice to replace this with the new call instead of deleting it

Reply:
No one is using this example. It ia also not complete, do we want to add all api calls to this?

Reply:
How do you know that no other developers or customers are using this (this is a public API)? We don't have all API calls, but the basic flow. create survey, add interviewer, add sample data, download survey and sample data...
If the example is not used, we should ask others to double check and fully delete the file. But I don't see the reason for deleting just that part, if we can still download survey data with the new endpoint

Reply:
If think so, I will add a download example.